app/controllers/content_items_controller.rb

```diff
@@ -23,8 +23,9 @@ def load_content_item
   def present(content_item)
     case content_item['format']
     when 'case_study' then CaseStudyPresenter.new(content_item)
-    when 'take_part' then TakePartPresenter.new(content_item)
     when 'statistics_announcement' then StatisticsAnnouncementPresenter.new(content_item)
+    when 'take_part' then TakePartPresenter.new(content_item)
+    when 'topical_event_about_page' then TopicalEventAboutPagePresenter.new(content_item)
```

Comment:
Much nicer :sparkles: 

`String#constantize` wraps the `Module.const_get` dance (but doesn't respect lexical context) if you go with that approach.